src/main/java/ch/uzh/ifi/hase/soprafs24/entity/Session.java

```diff
@@ -0,0 +1,35 @@
+package ch.uzh.ifi.hase.soprafs24.entity;
+
+
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.*;
+import java.io.Serializable;
+
+
+@Entity
+@Table(name = "SESSION")
+public class Session implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @Getter
+    @Setter
+    @GeneratedValue
+    private String sessionID;
```

Comment:
I think we shouldn't add here the @GeneratedValue decorator because we manually set this to a UUID. Please test after removing it, if it works as expected.